doc/sphinx/changes.rst

```diff
@@ -8,6 +8,667 @@ Recent changes
 
    .. include:: ../unreleased.rst
 
+Version 8.13
+------------
+
+Summary of changes
+~~~~~~~~~~~~~~~~~~
+
+Coq version 8.13 integrates many usability improvements, as well
+as extensions of the core language.
+The main changes include:
+
+  - Introduction of :ref:`primitive persistent arrays<primitive-arrays>`
+    in the core language, implemented using imperative persistent arrays.
+  - Introduction of definitional proof irrelevance for the equality type
+    defined in the SProp sort, using the :flag:`Definitional UIP` flag,
+    (deactivated by default). This models definitional uniqueness of
+    identity proofs for this type. It is deactivated by default as
+    it results in non-termination in combination with impredicativity.
+    Use of this flag is also shown by :cmd:`Print Assumptions`.
```

Comment:
```suggestion
    This flag is also used by :cmd:`Print Assumptions`.
```

Not sure if that's what you meant.  The "shown" doesn't read well.

Reply:
printed is better I think, fixed.